app/scripts/components/Share/share-template.html

```diff
@@ -1,17 +1,17 @@
 <div class="c-share">
   <h3 class="share-title">Share this page</h3>
   <div class="share-url-container">
-    <input readonly class="share-input" type="text" v-bind:value="sharedUrl">
-    <button class="c-button -filled" type="button">Copy</button>
+    <input readonly id="copyInput" class="share-input" type="text" v-bind:value="sharedUrl">
```

Comment:
`v-bind:value` its ok in this case cause it's a `readonly` input. But for other use cases vue accepts bidirectional data-binding with `v-model`. More info: https://vuejs.org/v2/guide/forms.html